book/content/introduction/introduction.md

```diff
@@ -53,6 +59,21 @@ Our core contributors are, in alphabetical order:
 
 You can see all of our incredible contributors in our [README](https://github.com/alan-turing-institute/the-turing-way#contributors) file, and screengrabbed below.
 
-![](../figures/Contributors.png)
+![Grid of pictures and names of project contributors](../figures/contributors-twopanel.png)
+
+### Citing _The Turing Way_
```

Comment:
Are you sure you want to maintain two copies of how to cite The Turing Way? It's also in README.md.

Reply:
yes, this is the introduction to the netlify book and many people just find that and not the github repo, so it will need to be in both places

Reply:
Yep - its an important point to consider but I think in this case we'll keep the two locations.